docs/reference/ml/df-analytics/apis/index.asciidoc

```diff
@@ -32,3 +35,7 @@ include::get-dfanalytics-stats.asciidoc[]
 //SET/START/STOP
 include::start-dfanalytics.asciidoc[]
 include::stop-dfanalytics.asciidoc[]
+//INFERENCE
+include::get-trained-model.asciidoc[]
```

Comment:
The rest of the APIs in this section are sorted alphabetically, so I think these should be distributed likewise (i.e. GET in with the other GET APIs, DELETE with the other deletes, etc)